sktree/tree/_unsup_tree.pxd

```diff
@@ -84,13 +86,22 @@ cdef class UnsupervisedTreeBuilder:
     # This class controls the various stopping criteria and the node splitting
     # evaluation order, e.g. depth-first or best-first.
 
-    cdef UnsupervisedSplitter splitter              # Splitting algorithm
+    cdef UnsupervisedSplitter splitter  # Splitting algorithm
 
     cdef SIZE_t min_samples_split       # Minimum number of samples in an internal node
     cdef SIZE_t min_samples_leaf        # Minimum number of samples in a leaf
     cdef double min_weight_leaf         # Minimum weight in a leaf
     cdef SIZE_t max_depth               # Maximal tree depth
     cdef double min_impurity_decrease   # Impurity threshold for early stopping
 
-    cpdef build(self, UnsupervisedTree tree, object X, cnp.ndarray sample_weight=*)
-    cdef _check_input(self, object X, cnp.ndarray sample_weight)
+    cpdef build(
+        self,
+        UnsupervisedTree tree,
+        object X,
+        cnp.ndarray sample_weight=*
```

Comment:
need to review this

Reply:
http://docs.cython.org/en/latest/src/userguide/pyrex_differences.html#optional-arguments-in-cdef-cpdef-functions

Reply:
ohh! Thank you :heart: